http-app-dependencies/pom.xml

```diff
@@ -29,6 +29,11 @@
 				<version>4.5.3</version>
 				<scope>test</scope>
 			</dependency>
+			<dependency>
+				<groupId>org.springframework.cloud.stream.app</groupId>
+				<artifactId>app-starters-security-common</artifactId>
+				<version>2.1.0.BUILD-SNAPSHOT</version>
```

Comment:
Why it is not in the `test` scope?
As far as we know this dependency is going to appear in all the generated apps. So, no need to specify it explicitly here.

If you would like to be sure in tests in this project before generation, then it must go to `test` scope.

Reply:
> As far as we know this dependency is going to appear in all the generated apps. 
This is not true! Because we found that the Management security in 2.1 + enables info and health by default, i've removed it from all generated apps. 
That is way i have to add it as non-test dependency for the http-source app.

Reply:
Ok the final state is that the dependency will be present in all app starters, but disabling the security only for Info, Health, LogFile and Bindings endpoints by default.

Still `test` scope makes perfect sense now.

Reply:
I still insist that we should not have these dependencies for testing in the BOM.

Just move the to the management in the root pom which is not going to be a part of target application when we import this BOM there.